Quadriga/src/main/java/edu/asu/spring/quadriga/web/resolver/EditResolverController.java

```diff
@@ -54,4 +55,21 @@ public String editResolver(Principal principal,
         return "redirect:/auth/resolvers";
     }
 
+    @RequestMapping(value = "/auth/resolvers/update", method = RequestMethod.POST, params = "test")
+    public String editResolverTest(Principal principal,
```

Comment:
If you're doing what I think you're doing with those two methods, they should be called through Ajax calls and not reload the whole page.